src/deprecated.jl

```diff
@@ -1,3 +1,162 @@
-Base.@deprecate_binding AbstractContext Assertion
-@deprecate Context(; limit=1.0, kwargs...) Threshold(limit)
-@deprecate WeightedContext(wv; limit=1.0, kwargs...) Threshold(limit; weights=wv)
+# Introduced in 0.6
+Base.@deprecate_binding(
+    AbstractContext,
+    Assertion,
+    false,
+    "Imputation contexts have been replaced with independent Impute.Assertion types.",
+)
+Base.@deprecate Context(; limit=1.0, kwargs...) Threshold(limit) false
+Base.@deprecate WeightedContext(wv; limit=1.0, kwargs...) Threshold(limit; weights=wv) false
```

Comment:
I am surprised this is non-breaking

Reply:
Oh. It is breaking.
In which case I don't think these deprecations make sense.

A section in release notes or docs on upgrading would allow to be more explicit.
Or a warning when you try and instantiate it.
Since the deprecation warning implies (at least to me) that for now it can be used identically

Reply:
👌